.github/workflows/main.yml

```diff
@@ -25,9 +25,6 @@ jobs:
       - name: Lint
         run: npm run lint
 
-      - name: Run local ganache chain in the background
-        run: (npm run chain&)
```

Comment:
Why is this not needed anymore?

Reply:
Ganache is started in code now via `ganache.provider()`, no need to start one externally.